environments/se/core/src/main/java/org/jboss/weld/environment/se/Weld.java

```diff
@@ -627,7 +629,7 @@ public Weld setClassLoader(ClassLoader classLoader) {
      * @param resourceLoader
      * @param bootstrap
      */
-    protected Deployment createDeployment(ResourceLoader resourceLoader, CDI11Bootstrap bootstrap) {
```

Comment:
I've seen the Weld class being subclassed more than once. Changing the signature of this protected method is likely to break those subclasses.